Client/Ecosia/Extensions/AppSettingsTableViewController+Ecosia.swift

```diff
@@ -153,8 +147,6 @@ extension AppSettingsTableViewController {
             CreateReferralCode(settings: self),
             AddReferral(settings: self),
             AddClaim(settings: self),
-            ChangeSearchCount(settings: self),
-            ResetSearchCount(settings: self),
```

Comment:
Given we still use the search count internally for experiments, I guess this debug settings are still useful for testing. What do you think about us keeping them?